src/main/resources/css/cucumber.css

```diff
@@ -210,6 +210,14 @@ tr:hover {
     color: #eee;
 }
 
+.white-space-normal {
+    white-space: normal;
```

Comment:
seems to be default behave http://www.w3schools.com/cssref/pr_text_white-space.asp so why do you need it?